website/src/components/variable/index.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
We could probably remove this and `Var` before merging, right?

Reply:
Yeah good call. I'll run through the components we added for versioning, and I'll either update this to include some actual tests, or remove it altogether.